gapic-generator-java-bom/pom.xml

```diff
@@ -29,6 +29,14 @@
         <type>pom</type>
         <scope>import</scope>
       </dependency>
+      <dependency>
```

Comment:
Not part of this PR. I think we could add the test jar to the [auth bom](https://github.com/googleapis/google-auth-library-java/blob/main/bom/pom.xml), so that it can be released as part of the above auth bom, then we should be able to remove this declaration. WDYT @suztomo ?